enterprise/server/cmd/ci_runner/main.go

```diff
@@ -1690,6 +1693,27 @@ func (ws *workspace) applyPatch(ctx context.Context, bsClient bspb.ByteStreamCli
 	return nil
 }
 
+var gitTroubles = []string{
+	"index.lock",
+	"shallow.lock",
+	"HEAD.lock",
+	"hooks/post-checkout",
+	"config.lock",
```

Comment:
how certain are we that it's safe to delete these files? could there be some background process running that's using these? And if we delete these lockfiles could those background processes wind up clobbering the git repo state?

Reply:
I'd prefer to wait for us having an example of this being an issue before checking this in. Maggie investigated the hang and didn't see any of these files.